### ISSUE TITLE
Add community telegram groups?

### ISSUE THREAD

**Issue by CasiniNift** [Polkadot-Blockchain-Academy/alumni-hub#182]
There are some interesting telegram groups in the Polkadot ecosystem. Perhaps they can be added in the "Ecosystem" section? 

Interesting Community Telegram Groups: 
- Fluid Treasury:
    - TLDR:  Alternative project funding mechanism through staking. 
    - Telegram: https://t.me/parachainstaking 
- Ambassadors Open Forum:
    - TLDR: A chat about the ambassador program’s changes/direction/goals. 
    - Telegram: https://t.me/ambxpert 
- OpenEVM 
    - TLDR: A chat about creating an EVM-compatible system chain on Polkadot (with DOT as the gas token). 
    - Telegram: https://t.me/openevm
